src/tonation.py

```diff
@@ -0,0 +1,19 @@
+import chord
+import copy
+
+
+class Tonation(chord.Chord):
+    def __init__(self, note, timestamp, duration, type):
+        super().__init__(note, timestamp, duration, type)
+        self.tonic = chord.Chord(note, timestamp, duration, type)
+        self.dominant = copy.deepcopy(self.tonic)
+        self.dominant.note += 7
+        self.dominant.type = "dur7"
+        self.subdominant = copy.deepcopy(self.tonic)
+        self.subdominant.note += 5
```

Comment:
afaiu this is purely 'guitaric' info. Maybe some way to mark this? Maybe Chord should be renamed to GuitarChord, to make it clear we don't want to use it anywhere else?